R/tables_spark.R

```diff
@@ -0,0 +1,73 @@
+tbl_quote_name <- function(name) {
+  x <- gsub("`", "``", name, fixed = TRUE)
```

Comment:
Yes, tried this with this name and this worked as expected. Borrowed this definition from DBI, I believe.